appyters/Gene_Age_Trends_Appyter/appyter.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
+    "name":"Gene_Age_Trends_Appyter",
+    "title": "Gene Expression vs Age Plot Generator",
+    "version": "0.0.1",
+    "description": "Generate Gene Expression vs Age Plots using cleaned RNA-seq data",
+    "authors": [
+      {
+        "name": "Tahmid Ahamed",
+        "email": "ahamedt@wharton.upenn.edu"
+      }
+    ],
+    "url": "https://github.com/MaayanLab/appyter-catalog",
+    "tags": [
+      "Gene Expression, Age, Mouse,Human"
+    ],
+    "license": "Apache-2.0",
```

Comment:
You say Apache-2.0 but have an MIT LICENSE file.. which one is it?

Reply:
My bad I copied the one I had in my private repo. Forgot to change the field to MIT.